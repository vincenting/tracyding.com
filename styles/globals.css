@tailwind base;
@tailwind components;
@tailwind utilities;

@font-face {
    font-family: "baby_doll";
    src: url("/fonts/baby_doll.woff2") format("woff2"), url("/fonts/baby_doll.woff") format("woff");
    font-weight: normal;
    font-style: normal;
}

@supports (-webkit-touch-callout: none) {
    .h-screen {
        /* The hack for Safari */
        height: -webkit-fill-available;
    }
}
