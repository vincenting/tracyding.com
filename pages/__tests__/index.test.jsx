import React from 'react'
import { describe, expect, it } from '@jest/globals'
import { shallow } from 'enzyme'
import Home from '../index.page'

describe('Home Page', () => {
  it('should render Home page as expected', () => {
    expect(shallow(<Home />)).toMatchSnapshot()
  })
})
