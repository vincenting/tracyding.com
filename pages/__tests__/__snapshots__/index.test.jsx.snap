// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render Home page as expected 1`] = `
<main
  className="w-screen h-screen overflow-hidden flex bg-orange-100"
>
  <div
    className="w-fit h-48 self-center ml-auto mr-auto container"
  >
    <h3
      className="text-lg text-center"
    >
      Dear visitor, I'm 
      <span
        className="text-pink-500 underline cursor-pointer"
        onClick={[Function]}
      >
        Tracy.
      </span>
       ðŸ˜Š
    </h3>
    <p
      className="mt-2"
    >
      My Portfolio Website is coming soon. 
    </p>
  </div>
</main>
`;
